asset_tracker_v2: Enable link time optimization (#11349)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

nRF9160: Asset Tracker v2
-------------------------

* Updated:

>   * Enabled link time optimization to reduce the flash size of the application. This can be disabled
>   by using the Kconfig option :kconfig:option:`CONFIG_ASSET_TRACKER_V2_LTO`.

Reviewer: ```suggestion
  * Enabled link time optimization to reduce the flash size of the application.
    You can disable this using the Kconfig option :kconfig:option:`CONFIG_ASSET_TRACKER_V2_LTO`.
```

nRF9160: Asset Tracker v2
-------------------------

* Updated:

>   * Enabled link time optimization to reduce the flash size of the application. You can disable this using the Kconfig option :kconfig:option:`CONFIG_ASSET_TRACKER_V2_LTO`.

Author: I see. Done. :)
